api/src/main/java/io/opentelemetry/trace/SpanId.java

```diff
@@ -65,46 +47,38 @@ public static int getSize() {
   }
 
   /**
-   * Returns the invalid {@code SpanId}. All bytes are 0.
+   * Returns the length of the base16 (hex) representation of the {@code SpanId}.
    *
-   * @return the invalid {@code SpanId}.
-   * @since 0.1.0
+   * @since 0.8.0
    */
-  public static SpanId getInvalid() {
-    return INVALID;
+  public static int getBase16Length() {
```

Comment:
Hex? If you used hex in SpanContext method names.

Reply:
yes! should fix this. good catch.

Reply:
doen